pages/linux/i3lock.md

```diff
@@ -0,0 +1,24 @@
+# i3lock
+
+> Simple screen locker built for the i3 window manager.
+> More information: <https://i3wm.org/i3lock>.
+
+- Lock screen with a simple color background (rrggbb format):
+
+`i3lock -c {{0000ff}}`
+
+- Lock screen to a PNG background:
```

Comment:
Does it have to be PNG, or it can be some other format too?

Reply:
It does have to be a PNG, unfortunately. There is a `--raw=format` flag which can be used to read a raw image instead of PNG, like this (taken from i3lock's man page):

`convert wallpaper.jpg RGB:- | i3lock --raw 3840x2160:rgb --image /dev/stdin`

I did not include this flag in the tldr entry as it is too specific and might end up confusing users.